package.json

```diff
@@ -6,7 +6,18 @@
         "dev": "next dev",
         "build": "next build",
         "start": "next start",
-        "lint": "eslint --fix --ext .ts,.tsx ./src"
+        "lint": "eslint --fix --ext .ts,.tsx ./src",
+        "release": "standard-version",
+        "release:minor": "standard-version --release-as minor",
+        "release:patch": "standard-version --release-as patch",
+        "release:major": "standard-version --release-as major"
+    },
+    "standard-version": {
+        "skip": {
+            "bump": true,
+            "commit": true,
+            "tag": true
```

Comment:
Let me know if you prefer any different configuration here, eg. to automatically tag or bump up based on the script used

Reply:
Nah, it's cool.